docusaurus/docs/dev-docs/migration/v4-to-v5/breaking-changes/server-default-log-level.md

```diff
@@ -0,0 +1,57 @@
+---
+title: Strapi Server log level configuration
+description: Strapi 5 allows users greater control over the level of detail in of their server logs
+displayed_sidebar: devDocsMigrationV5Sidebar
+tags:
+ - breaking changes
```

Comment:
I think you can also add the `server` tag since the configuration is both for middlewares and server :)

Reply:
Okay, I'll do that now.